src/SelectValidator.jsx

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable */
 import React from 'react';
 import Select from 'material-ui/Select';
-import { FormHelperText } from 'material-ui/Form';
+import { FormControl, FormHelperText } from 'material-ui/Form';
```

Comment:
You can see below that `formControl: FormControl,` is taken from props

Reply:
Yeah you're right, sorry
I'm facing an error around this component